app/helpers/pdf_js_helper.rb

```diff
@@ -5,6 +5,11 @@ def pdf_js_url(path)
     "/pdf.js/viewer.html?file=#{path}##{query_param}"
   end
 
+  def pdf_file_set_presenter(file_set_presenters)
+    # currently only supports one pdf per work
+    file_set_presenters.select(&:pdf?).first
```

Comment:
is it ok that this could return nil?

Reply:
hmm, it shouldn't return `nil` because it wouldn't should get called if there isn't any pdf file sets

```erb
<% elsif Flipflop.default_pdf_viewer? && presenter.show_pdf_viewer? && presenter.file_set_presenters.any?(&:pdf?) %>
  <%= render 'pdf_js', file_set_presenter: pdf_file_set_presenter(@presenter.file_set_presenters) %>
```

Reply:
ah, missed that. ok thanks!